snap.cabal.orig

```diff
@@ -0,0 +1,28 @@
+Name:                snap
```

Comment:
What's the purpose of this file?

Reply:
@dbalseiro Mainly for learning purposes, how the changes should end when comparing the initial file from the scratch with the final result.